migrations/schema.sql

```diff
@@ -828,7 +828,8 @@ CREATE TABLE public.groups (
     num_members integer,
     slack_hook_url text,
     slack_team text,
-    slack_configure_url text
+    slack_configure_url text,
+    "type" text DEFAULT 'group'::text
```

Comment:
ohh is it because we can only search text fields? that seems off, we really dont need a full text field for the type.

Reply:
no, I just picked the wrong knex helper function